lib/tessel/wifi.js

```diff
@@ -12,7 +12,9 @@ var Tessel = require('./tessel');
 
 Tessel.prototype.findAvailableNetworks = function() {
   logs.info('Scanning for available networks...');
-  return this.simpleExec(commands.scanWiFi())
+  var scanWiFi = () => this.simpleExec(commands.scanWiFi());
+  return this.simpleExec(commands.turnOnWifi(true))
+    .then(scanWiFi)
```

Comment:
I feel that it's unnecessary to store the function in a variable, how about just adding the following?

```javascript
.then(() => this.simpleExec(commands.scanWiFi()))
```